hw/ip/pwm/data/pwm.hjson

```diff
@@ -31,10 +31,10 @@
     }
   ],
   registers: [
-    { name: "REGEN",
+    { name: "REGWEN",
       desc: "Register write enable for all control registers",
-      swaccess: "rw1c",
-      hwaccess: "hro",
+      swaccess: "rw0c",
+      hwaccess: "none",
       async: "clk_core_i",
```

Comment:
sorry, this is on me for not having better checks on the reggen side, i'll look into adding it. But you don't need to have the `async` field for regwens.  The regwen is actually applied 'pre' domain crossing, and the logic underneath will automatically handle it.

Reply:
Thanks a lot for your comments! I have removed the async field!

Reply:
Great.  Thanks for the feedback @tjaychen!  Now that Muqing has removed this is it GTG in your opinion?

Muqing, I see you removed this.  When it is all passed